BRAINSCut/cnpy/cnpy.cpp

```diff
@@ -88,8 +88,8 @@ void cnpy::parse_npy_header(FILE* fp, unsigned int& word_size, unsigned int*& sh
     //byte order code | stands for not applicable. 
```

Comment:
Might want to pull in the upstream changes? They are using `reinterpret_cast` already, so the warnings should go away. https://github.com/rogersce/cnpy/blob/master/cnpy.cpp